src/plugins/gs-plugin-external-apps.c

```diff
@@ -43,8 +44,15 @@
 
 #define TMP_ASSSETS_PREFIX "gs-external-apps"
 
+enum GsPluginExternalType {
+	GS_PLUGIN_EXTERNAL_TYPE_DEB = 1,
+	GS_PLUGIN_EXTERNAL_TYPE_TAR,
+	GS_PLUGIN_EXTERNAL_TYPE_UNKNOWN,
```

Comment:
Why would you want to test it like that? I don't think enums should be tested with < or > .
Mind the comma too.